service/src/main/resources/landingzone/aks/configmap/ContainerLogV2.yaml

```diff
@@ -0,0 +1,209 @@
+kind: ConfigMap
```

Comment:
Yes. That's like template from the documentation. I think we don't need the whole document, since we need only one option. I will validate it and possibly update it to have only what's needed.